### ISSUE TITLE
Polygon should support MultiPolygon format

### ISSUE THREAD

**Issue by aaron-kaplan** [thedirtyfew/dash-leaflet#110]
Leaflet's Polygon class supports a shape description that's either
- a list of points, where a point is a list of the form [lat, lon]
- a list of rings, where a ring is a list of points (as defined above)
- a list of polygons, where a polygon is a list of rings (as defined above).
In the last case, the first ring of a polygon represents the outer ring of the shape, and the remaining rings represent holes. Excerpt from the Leaflet [docs](https://leafletjs.com/reference-1.7.1.html#polygon):
```
Additionally, you can pass a multi-dimensional array to represent a MultiPolygon shape.

var latlngs = [
  [ // first polygon
    [[37, -109.05],[41, -109.03],[41, -102.05],[37, -102.04]], // outer ring
    [[37.29, -108.58],[40.71, -108.58],[40.71, -102.50],[37.29, -102.50]] // hole
  ],
  [ // second polygon
    [[41, -111.03],[45, -111.04],[45, -104.05],[41, -104.05]]
  ]
];
```

react-leaflet [supports](https://github.com/PaulLeCam/react-leaflet/blob/master/packages/react-leaflet/src/Polygon.tsx#L11) all three cases:
```
export interface PolygonProps extends PolylineOptions, PathProps {
  children?: ReactNode
  positions: LatLngExpression[] | LatLngExpression[][] | LatLngExpression[][][]
}
```
but dash-leaflet unnecessarily excludes the "MultiPolygon" case, only [allowing](https://github.com/thedirtyfew/dash-leaflet/blob/f8823bca5f7afb0fdc99c80d5105485b72d8a8e8/src/lib/components/Polygon.react.js#L21-L24) the first two cases:
```
    positions: PropTypes.oneOfType([
        PropTypes.arrayOf(PropTypes.arrayOf(PropTypes.number)),
        PropTypes.arrayOf(PropTypes.arrayOf(PropTypes.arrayOf(PropTypes.number)))
    ]).isRequired,
```

**Comment by emilhe** [comment]
Did you try it out? Unless you use debug mode, the type definitions are not strictly enforced.

**Comment by aaron-kaplan** [comment]
You're right. I was in debug mode when I observed this problem, and it goes away in production mode.

Shouldn't it be fixed anyway?

**Comment by emilhe** [comment]
Jep, it should! It was just a suggestion to get you going :)

**Comment by emilhe** [comment]
Should be fixed in 0.1.21 release.
